docs/release_process.md

```diff
@@ -247,11 +267,150 @@ git merge stabilization-vX.Y.Z
     Regards,
     ```
 
-# Downstream Build (Fedora)
+# Downstream Build
+
+It is a good for the community when the packages are also updated and released with the latest
+stable release. In this section there is information about how to update the packages in specific
+distros.
+## Fedora
```

Comment:
For me it makes a lot of sense to have this step by step here. In case of Fedora, updating the packages when a new Upstream release is out is directly connected in the process.

Therefore, it is much better to have the relevant steps here than only pointing the users to Fedora documentation where there is indeed a rich documentation but not everything is relevant for the context of scap-security-guide package context. These steps here are handful when updating the packages.

Also, if packages in other distros are also updated based on the latest Upstream Releases, contributions including their steps here are welcome.

Reply:
Still, the upstream release is a separate process from Fedora, and other distros downstream builds.
My main suggestion here was to separate the Fedora Guide into a separate doc.
What do you think about moving them to a separate file and just linking them from the upstream release guide?